tsl/src/nodes/vector_agg/grouping_policy_hash.c

```diff
@@ -275,14 +278,14 @@ add_one_range(GroupingPolicyHash *policy, DecompressBatchState *batch_state, con
 }
 
 static void
-gp_hash_add_batch(GroupingPolicy *gp, DecompressBatchState *batch_state)
+gp_hash_add_batch(GroupingPolicy *gp, TupleTableSlot *vector_slot)
 {
 	GroupingPolicyHash *policy = (GroupingPolicyHash *) gp;
+	uint16 n;
+	const uint64 *restrict filter = vector_slot_get_qual_result(vector_slot, &n);
```

Comment:
Maybe something more elaborate than `n`?

```suggestion
	uint16 row_count;
	const uint64 *restrict filter = vector_slot_get_qual_result(vector_slot, &row_count);
```